smartfile/__init__.py

```diff
@@ -1,16 +1,17 @@
-from netrc import netrc
 import os
 import re
+import shutil
 import time
 import urllib
+
+from netrc import netrc
```

Comment:
NOTE TO SELF: This library is used to import confuration that usually contains FTP usernames and passwords. 
Source: https://www.safaribooksonline.com/library/view/python-standard-library/0596000960/ch05s09.html